package/batocera/emulationstation/batocera-es-system/roms/model2/_readme.txt

```diff
@@ -14,7 +14,7 @@ Controller configuration
 
 Note: A keyboard & mouse is required in addition to your controller(s).
 
-You must map your controls per game ROM you intend to run.
+A set on inout files is provided however you may want to map your controls per game ROM you intend to run.
```

Comment:
Typo here, "A set of input files" maybe? 
That's a great addition, by the way! Thank you!

Reply:
Fixed!